modAionImpl/src/org/aion/zero/impl/blockchain/AionHub.java

```diff
@@ -133,7 +133,7 @@ private void initializeHub(
         }
 
         this.startingBlock = this.blockchain.getBestBlock();
-        if (!cfg.getConsensus().isSeed()) {
+        if (!cfg.getTx().isSeedMode()) {
```

Comment:
This change will be problematic when migrating to the release. I am not sure if it's worth the trouble that it may cause. Can we introduce some sort of config migration to avoid the need for manual user update?

Reply:
No, because the seed mode is a hidden config option. So I don't think it will have an issue for migrating.

Reply:
I can keep the hidden config option in the cfgConsesus (and add comments), then update the logic to CfgTx